app/src/main/java/com/example/playlistmaker/media/addPlayList/presention/ui/PlaylistAdapter.kt

```diff
@@ -0,0 +1,83 @@
+package com.example.playlistmaker.media.addPlayList.presention.ui
+
+import android.os.Environment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
+import com.bumptech.glide.load.resource.bitmap.RoundedCorners
+import com.bumptech.glide.request.RequestOptions
+import com.example.playlistmaker.R
+import com.example.playlistmaker.media.domain.models.PlaylistModel
+
+import java.io.File
+
+class PlaylistAdapter(val listenerItemPlaylist: ClickListener, private val isVertical: Boolean) :
```

Comment:
```suggestion
class PlaylistAdapter(private val listenerItemPlaylist: ClickListener, private val isVertical: Boolean) :
```
Чтобы не было доступа извне к этому листенеру